docs/service-discovery/static-list.md

```diff
@@ -0,0 +1,29 @@
+# Static List Service Discovery
+
+In some situations, such as demos, development, or testing, you may want to mock the service discovery by providing a predefined list of service instances.
+For this purpose, Stork comes with a `static` service discovery type.
+
+## Dependency
+
+To use the `static` service discovery, first add the appropriate Service Discovery provider dependency to your project:
+
+```xml
+<dependency>
+    <groupId>io.smallrye.stork</groupId>
+    <artifactId>stork-service-discovery-static-list</artifactId>
+    <version>{{version.current}}</version>
+</dependency>
+```
+
+## Configuration
+
+For each service that should use the static list of service instances configure the service discovery `type`:
+
+```properties
+stork.my-service.service-discovery.type=static
```

Comment:
You should mention the address-list, as it's mandatory.

Reply:
This is how this page is rendered, I didn't want to duplicated the info on it being mandatory but I'm fine adding it
![image](https://user-images.githubusercontent.com/3901322/159934679-77f1055e-2c6c-433b-9e74-219e9b051eb5.png)